### DISCUSSION TITLE
Can I apply to GCE while not being a Undergraduate

### DISCUSSION THREAD

**Discussion by shaynzz** [community/community#37228] [GitHub Education] [answered]
Please lmk

**Comment by fykaa** [comment, marked as answer]
Hi @shaynzz 
Thanks for Asking
Github Campus Expert is specially meant for Undergrads and hence you must be enrolled for a Undergrad education.
Hope this answers your question../

**Comment by shaynzz** [comment]
Oh OKayeeeeeeee thanks

**Comment by shaynzz** [comment]
But can I APply if I've just completed my undergraduation

**Comment by fykaa** [comment]
@shaynzz You are required to be registered for the year you are applying in.
